### PR TITLE
removed duplicate allow and empty lines and spaces

### PR DESCRIPTION
**Pull request by unique-usernames-are-annoying**

### CODE REVIEW

File: CleanSlate/common/scripted_triggers/00_china_triggers.txt
Comment (schwarherz) [inline CleanSlate/common/scripted_triggers/00_china_triggers.txt:1]: Encoding seems to have changed on this file. Need to make sure it's corrected back to Windows-1252

File: CleanSlate/events/base_religious_events.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #
 ###################################
 
-# Written by Henrik F�hraeus, Johan Andersson & Niklas Strid
+# Written by Henrik F�hraeus, Johan Andersson & Niklas Strid
```
Comment (schwarherz) [inline CleanSlate/events/base_religious_events.txt:9]:
```suggestion
# Written by Henrik Fåhraeus, Johan Andersson & Niklas Strid
```

File: CleanSlate/events/sunset_invasion.txt [hunk 2]
```diff
@@ -5,7 +5,7 @@
 #
 ###################################
 
-# Written by Henrik F�hraeus and Henrik Eklund
+# Written by Henrik F�hraeus and Henrik Eklund
```
Comment (schwarherz) [inline CleanSlate/events/sunset_invasion.txt:8]:
```suggestion
# Written by Henrik Fåhraeus and Henrik Eklund
```

**Review by schwarherz — CHANGES_REQUESTED** [review]
It looks like the encoding may have been changed in several files. I want to verify that the encoding is correct (Windows-1252) and that any malformed characters are corrected before approval. Otherwise the vast majority of these changes are whitespace-only and should be fine.

File: CleanSlate/common/scripted_triggers/00_china_triggers.txt [hunk 3]
```diff
@@ -209,7 +209,7 @@ china_offmap_decisions_enabled_trigger = {
 		NOT = { has_policy = china_isolationist }
 
 		custom_tooltip = {
-			text = china_offmap_decisions_enabled_trigger_war_tt # "Is at peace with §Y[From.GetTitledFirstName]§!"
+			text = china_offmap_decisions_enabled_trigger_war_tt # "Is at peace with �Y[From.GetTitledFirstName]�!"
```
Comment (schwarherz) [inline CleanSlate/common/scripted_triggers/00_china_triggers.txt:212]:
```suggestion
			text = china_offmap_decisions_enabled_trigger_war_tt # "Is at peace with §Y[From.GetTitledFirstName]§!"
```

File: CleanSlate/common/scripted_triggers/00_china_triggers.txt [hunk 4]
```diff
@@ -273,7 +273,7 @@ china_offmap_hostile_decisions_enabled_trigger = {
 
 	offmap_china = {
 		custom_tooltip = {
-			text = china_offmap_decisions_enabled_trigger_war_tt # "Is at peace with §Y[From.GetTitledFirstName]§!"
+			text = china_offmap_decisions_enabled_trigger_war_tt # "Is at peace with �Y[From.GetTitledFirstName]�!"
```
Comment (schwarherz) [inline CleanSlate/common/scripted_triggers/00_china_triggers.txt:276]:
```suggestion
			text = china_offmap_decisions_enabled_trigger_war_tt # "Is at peace with §Y[From.GetTitledFirstName]§!"
```

File: CleanSlate/common/scripted_triggers/00_china_triggers.txt
Comment (unique-usernames-are-annoying) [reply CleanSlate/common/scripted_triggers/00_china_triggers.txt:1]: I looked at the 3 files mentioned, however when i open it either with the vscode setting "files.encoding": "windows1252" it gets shown properly and opening it without it, it gets opened with utf8 and reopening it with windows1252 shows everything correct. Though this is normal for Github desktop to show special characters encoded in windows1252 wrong, which is also the reason i suggested the settings you are currently trying to add in the other PR.

**Comment by schwarherz** [comment]
Created new branch on origin for these so that I could rebase the request onto main

**Pull request closed without merging**